intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -2903,10 +2903,78 @@ def scan_exchange_identifier(field):
     }
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-amqp-report/
+accessible_amqp = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port')
+    ],
+    'optional_fields': [
+        ('protocol.transport', 'protocol'),
+        ('source.reverse_dns', 'hostname'),
+        ('source.asn', 'asn', invalidate_zero),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('extra.', 'tag'),
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'channel', convert_int),
+        ('extra.', 'message_length', convert_int),
+        ('extra.', 'class', convert_int),
+        ('extra.', 'method', convert_int),
+        ('extra.', 'version_major', validate_to_none),
+        ('extra.', 'version_minor', validate_to_none),
+        ('extra.', 'capabilities', validate_to_none),
+        ('extra.', 'cluster_name', validate_to_none),
+        ('extra.', 'platform', validate_to_none),
+        ('extra.', 'product', validate_to_none),
+        ('extra.', 'product_version', validate_to_none),
+        ('extra.', 'mechanisms', validate_to_none),
+        ('extra.', 'locales', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'other',
+        'classification.type': 'other',
+        'classification.identifier': 'accessible-amqp',
+    }
+}
+
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-amqp-report/
+device_id = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port')
+    ],
+    'optional_fields': [
+        ('protocol.transport', 'protocol'),
+        ('source.reverse_dns', 'hostname'),
+        ('source.asn', 'asn', invalidate_zero),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('extra.', 'tag'),
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'sector', validate_to_none),
+        ('extra.', 'device_vendor', validate_to_none),
+        ('extra.', 'device_type', validate_to_none),
+        ('extra.', 'device_model', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'other',
+        'classification.type': 'other',
```

Comment:
We can also use `undetermined` here. The RSIT definition of undetermined is `The categorisation of the incident is unknown/undetermined.`

Reply:
Yes, undetermined may be more appropriate since it is only informational